packages/open-payments/src/client/token.ts

```diff
@@ -0,0 +1,41 @@
+import { HttpMethod } from 'openapi'
+import { RouteDeps } from '.'
+import { getASPath, InteractiveGrant, NonInteractiveGrant } from '../types'
+import { post } from './requests'
+
+export interface TokenRouteDeps extends RouteDeps {
+  client: string
+}
+
+interface RotateRequestArgs {
+  url: string
+}
+
+export interface TokenRoutes {
+  rotate(
+    args: RotateRequestArgs
+  ): Promise<InteractiveGrant | NonInteractiveGrant>
+}
+
+export const createTokenRoutes = (deps: TokenRouteDeps): TokenRoutes => {
+  const revokeTokenValidator = deps.openApi.createResponseValidator<
+    InteractiveGrant | NonInteractiveGrant
```

Comment:
```suggestion
    AccessToken
```

I think we should create a `AccessToken` type to export, and have that be the return type here. This is what we get back in the response. You can see it in the open api spec: https://github.com/interledger/open-payments/blob/ae0e924a56fe9f12834f65da109a1042760c1605/openapi/auth-server.yaml#L247-L253

Similarly, if you take a look at the incoming request, there is no `body` parameter. Just the `url` provided in the `post` call is sufficient for now.